shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java

```diff
@@ -135,4 +135,15 @@ public String getDescription() {
         + "under storage will be returned. An administrator should then reconcile the differences."
         + "Specify -r to repair the inconsistent files.";
   }
+
+  @Override
+  public boolean validateArgs(String... args) {
+    boolean valid = args.length <= getNumOfArgs();
```

Comment:
Based on my suggestion, we don't need to modify `validateArgs` here and leave it to inherit from super class